config/samples/backends/dell/powermax/iscsi/cinder-volume-powermax-secrets.yaml

```diff
@@ -0,0 +1,17 @@
+# Define the "cinder-volume-powermax-secrets" Secret that contains sensitive
+# information pertaining to the [powermax-iscsi] backend.
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    service: cinder
+    component: cinder-volume
+  name: cinder-volume-powermax-secrets
+type: Opaque
+stringData:
+  powermax-secrets.conf: |
+    [powermax-iscsi]
+    san_ip = 10.225.104.114
+    san_login = smc
+    san_password = smc
+
```

Comment:
CI is failing because of this empty line at the end of the file: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openstack-k8s-operators_cinder-operator/331/pull-ci-openstack-k8s-operators-cinder-operator-main-precommit-check/1760003851160653824